app/src/pages/inside/testCaseLibraryPage/testCaseDetailsPage/testCaseDetailsPage.tsx

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2025 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useState, useCallback } from 'react';
+import { useIntl } from 'react-intl';
+import classNames from 'classnames/bind';
+import { Pagination } from '@reportportal/ui-kit';
+import { TestCaseList } from 'components/testCaseList';
+import { ITEMS_PER_PAGE_OPTIONS } from 'components/testCaseList/mockData';
+import {
+  DEFAULT_CURRENT_PAGE,
+  DEFAULT_ITEMS_PER_PAGE,
+} from 'components/testCaseList/testCaseList.constants';
+import { TestCase } from 'components/testCaseList/types';
+import { messages } from '../messages';
+import styles from './testCaseDetailsPage.scss';
+
+const cx = classNames.bind(styles);
+
+interface TestCaseDetailsPageProps {
+  testCases: TestCase[];
+  loading: boolean;
+  searchValue: string;
+  setSearchValue: (value: string) => void;
+  deleteTestCase: (id: string | number) => void;
+  duplicateTestCase: (testCase: TestCase, copySuffix: string) => void;
+}
+
+export const TestCaseDetailsPage = ({
```

Comment:
It looks suspicious name for me since it renders list. I do believe that the page that I work on is intended to be testCaseDetailsPage:
https://www.figma.com/design/e8vBoLmhDhs1fxrcFYpxdJ/RP6-Test-Case-Library?node-id=3950-182333&t=5hsvqcccNNdDEvvN-1